docs/client/logging_in.rst

```diff
@@ -0,0 +1,54 @@
+.. py:currentmodule:: coc
+
+Logging In
+==========
+A coc.py client instance can be created directly by :class:`Client`. The instance can then be logged in with the methods
+
+.. autofunction:: coc.client.Client.login
+
+.. autofunction:: coc.client.Client.login_with_tokens
+
+
+Example
+~~~~~~~
+.. code-block:: python3
```

Comment:
We should also add an example of using the context manager

Reply:
huh
Isn't this exact example one for using the context manager?